tasks/bundle_util.rb

```diff
@@ -91,4 +93,23 @@ def with_gemfile(gemfile)
   def platforms
     PLATFORMS.keys
   end
+
+  def validate_bundler_version!
+    actual_version = `#{Gem.ruby} #{Gem.bin_path("bundler", "bundle")} -v`.chomp
+    unless actual_version.end_with?(desired_bundler_version)
+      raise "Running #{actual_version}. Please install the required"\
```

Comment:
might call actual_version something like `version_string` since it looks like `Bundler version 1.11.2`